### DISCUSSION TITLE
Html Map Tag

### DISCUSSION THREAD

**Discussion by RNovachkov** [tailwindlabs/tailwindcss#8323] [Help] [answered]
**What version of Tailwind CSS are you using?**

https://play.tailwindcss.com (with the default settings, only the html is different)
tailwindcss v3.0.24

**What browser are you using?**

Opera

**What operating system are you using?**

Windows

**Reproduction URL**
(Html content in the editor)
`<h2>Hello world</h2>
<img src="https://images.unsplash.com/photo-1650138318699-a7916aced967?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80" usemap="mymap" />
<map name="mymap">
  <area shape="rect" title="Hello world" coords="0,770,0,1060" href="/#drd"/>
</map>`

**Describe your issue**

Map Tag not working with tailwindcss

**Comment by thecrypticace** [comment, marked as answer]
This is not Tailwind CSS specific. There's a few things you'll need to fix here:
1. On the `<img>` tag the `usemap` attribute should be `#mymap`: `usemap="#mymap"`
2. Your coordinates on `<area>` are flipped. The coordinates are the four corners of the box as x1, y1, x2, y2 and should be: `0,0,770,1060` instead. The coordinates `0,770,0,1060` describe a box that is 0px wide on the left side of the image and 300px tall which is why you can't click anything.
3. Use the `alt` attribute on `<area>` elements — not `title`

Hope that helps!

**Reply by RNovachkov** [reply]
Thanks
